ignite/handlers/param_scheduler.py

```diff
@@ -1393,6 +1393,133 @@ def simulate_values(cls, num_events: int, schedulers: List[_LRScheduler], **kwar
             return values
 
 
+class ReduceLROnPlateauScheduler(ParamScheduler):
+    """Reduce LR when a metric stops improving.
+    Wrapper of torch.optim.lr_scheduler.ReduceLROnPlateau .
+    Args:
+        optimizer (Optimizer): Wrapped optimizer.
```

Comment:
If this docs are just copied from pytorch docs, let's remove it and provide the link to it.

Reply:
Hi, No. Parameters `metric_name`, `trainer`, `save_history` and `param_group_index` are not there. The example also is not from there.

Reply:
I see, then we can remove type hints from docsting as they will be fetched automatically from typehints from code

Reply:
Other parameters are copied from there. You mean to remove just those params?

Reply:
No, i was talking only about type hints